Tests/RunnerPart2/Runner2Test.php

```diff
@@ -250,6 +252,183 @@ public function testComponentRunsIfSandboxProjectDoesNotExist(): void
         self::assertNotContains('Environment "AZURE_STORAGE_CONNECTION_STRING"', $containerOutput);
     }
 
+    public function testArtifactsUpload()
+    {
+        $storageApiMock = $this->getMockBuilder(StorageApiClient::class)
+            ->setConstructorArgs([[
+                'url' => STORAGE_API_URL,
+                'token' => STORAGE_API_TOKEN,
+            ]])
+            ->onlyMethods(['verifyToken'])
+            ->getMock()
+        ;
+        $storageApiMock->method('verifyToken')->willReturn([
+            'owner' => [
+                'id' => '1234',
+                'fileStorageProvider' => 'local',
+                'features' => ['artifacts'],
```

Comment:
Dobrej hack, to me nikdy nenapadlo

Reply:
No jinde pouzivame token s tou featurou.

Reply:
Som to opisal z ineho testu ;)